jdbc-ucp/src/test/groovy/io/micronaut/configuration/jdbc/ucp/DatasourceConfigurationSpec.groovy

```diff
@@ -203,7 +204,7 @@ class DatasourceConfigurationSpec extends Specification {
         dataSource.getMaxPoolSize() == 20
         dataSource.getTimeoutCheckInterval() == 5
         dataSource.getInactiveConnectionTimeout() == 10
-        dataSource.getConnectionWaitTimeout() == 10
+        dataSource.getConnectionWaitDuration() == Duration.ofSeconds(3)
```

Comment:
Have the settings' default changed?

Reply:
Defaults were 3 (seconds) before but the test was overriding to 10. Since the property is gone then I have changed test to expect default values. Now I have updated test to update new property `connectionWaitDuration` to `10s` and then expect that value.